examples/multiphysics/rising-bubble/rising-bubble.py

```diff
@@ -329,8 +353,11 @@
 ax3.set_xlabel(r'Rising time [T]')
 ax3.ticklabel_format(useOffset=False, style='plain', axis='y')
 ax3.legend(loc="upper left")
-fig3.savefig(f'./global-mass-conservation-case' + str(case_number) + '.png',dpi=300)
-plt.show()
+if (args.validate):
+  fig3.savefig(f'./global-mass-conservation-case' + str(case_number) + '.pdf')
```

Comment:
For the first 3 figures, the savetxt is also called. Is it missing for fig3 and fig4?

Reply:
For now I'd rather not monitor that, but it can be added in the future :)